plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -250,7 +250,10 @@ export function stateModelFactory(pluginManager: PluginManager) {
          * #property
          * highlights on the LGV from the URL parameters
          */
-        highlight: types.maybe(types.frozen<Required<ParsedLocString>>()),
+        highlight: types.optional(
```

Comment:
this might break sessions that use the old method. might need a preProcessSnapshot to load the old version

Reply:
let me know if I did this right:

```
    .preProcessSnapshot(snap => {
      const { highlight, ...rest } = snap
      return {
        highlight: highlight ? [...highlight] : [],
        ...rest,
      }
    })
 ```

Reply:
i believe it would want to check if snap.highlight is an array, and then optionally convert it into an array. that would then support the old formats (converts into an array) and new formats (when snap.highlight is already an array or undefined)